frontend/src/assets/css/variables.scss

```diff
@@ -36,9 +36,6 @@
   --z-index-level-one: 100;
   --z-index-tooltip: 1000;
   --achievement-columns: 3;
-
-  /* header height */
-  --header-height: 105px;
```

Comment:
@PablloZz @andanf-e Is it a good idea to delete this variable?

Reply:
If it won't be used, then yes